modules/bmo-editing-the-hostfirmwarecomponents-resource.adoc

```diff
@@ -0,0 +1,132 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+[id="editing-the-hostfirmwarecomponents-resource_{context}"]
+= Editing the HostFirmwareComponents resource
+
+You can edit the `HostFirmwareComponents` resource of a node.
+
+.Procedure
+
+. Get the detailed list of `HostFirmwareComponents` resources:
++
+[source,terminal]
+----
+$ oc get hostfirmwarecomponents -n openshift-machine-api -o yaml
+----
+
+. Edit a host's `HostFirmwareComponents` resource:
++
+[source,terminal]
+----
+$ oc edit <host_name> hostfirmwarecomponents -n openshift-machine-api
+----
++
+Where `<host_name>` is the name of the host. The `HostFirmwareComponents` resource will open in the default editor for your terminal.
```

Comment:
```suggestion
The `HostFirmwareComponents` resource opens in the default editor for your terminal.
```

Suggest using a callout for <host_name> to avoid having this sentence fragment.